src/singletons/draw-manager/earth.ts

```diff
@@ -651,7 +651,7 @@ export class Earth {
 
         //................................................
         // Specular lighting
-        vec3 specLightColor = texture(uSpecMap, vUv).rgb * diffuse * 0.1;
+        vec3 specLightColor = texture(uSpecMap, vUv).rgb * diffuse * 0.2;
```

Comment:
This will illuminate parts of the map that are in sunlight - very noticeable with the deserts in Africa. Lowering it will make other parts appear brighter (relatively).